web/src/main/java/com/softserve/osbb/dto/adapter/EntityToDTOAdapter.java

```diff
@@ -1,3 +1,9 @@
+/*
+ * Project “OSBB” – a web-application which is a godsend for condominium head, managers and 
```

Comment:
do I see this javadoc copy pasted everywhere ?
if so, you'd better withe package-info.java in your package and place this javadoc there.
no need to copy paste the same javadoc everywhere around :(